fog/sql_recovery_db/src/lib.rs

```diff
@@ -247,11 +247,13 @@ impl RecoveryDb for SqlRecoveryDb {
     fn get_ingress_key_records(
         &self,
         start_block_at_least: u64,
+        should_include_lost_keys: bool,
```

Comment:
Multiple bool arguments makes it difficult to read the code. I like what you did with the comment inside the assert statements below to help with that, but I'm wondering if the filtering is necessary at all given how this is used. 
I believe the `start_block_at_least` filter is there because this helps with fog-view performance, but since the other two new filters are only used by manual cli client calls, I am not sure we need to optimize for returning less data in that path.
With that said I'm okay leaving it as is if you or anyone else thinks differently.

Reply:
Agreed that boolean parameters decreases readability- a solution is to make a struct that contains named boolean fields. 

That said, @garbageslam  mentioned that it might be nice to give CLI users (i.e. the ops team) the ability to pass flags that filter based on retired / lost key status. I'm open to either way, but slightly prefer leaving it as I think it would save the ops team from performing the additional step of having to filter the results themselves? @garbageslam Any thoughts on this?

Reply:
Another option would be to make the function take a struct, so that the parameters are named?